dist/kothic-leaflet.js

```diff
@@ -135,5 +141,20 @@ L.TileLayer.Kothic = L.TileLayer.Canvas.extend({
         script.charset = 'utf-8';
         document.getElementsByTagName('head')[0].appendChild(script);
         return script;
+    },
+
+    _loadJSON: function(url, zoom, x, y) {
+        var xhr = new XMLHttpRequest();
+        xhr.onreadystatechange = function() {
+            if (xhr.readyState == XMLHttpRequest.DONE) {
+                if (xhr.status == 200) {
+                    window.onKothicDataResponse(JSON.parse(xhr.responseText), zoom, x, y);
```

Comment:
It looks like everything is working fine if I call this._onKothicDataResponse() at this place. However, nothing is render if I remove the bind in line 50, if it is that what you had in mind. My JavaScript-foo is too bad to diagnose what is happening there.